### ISSUE TITLE
Task reminders are discarded upon reboot and can be closed

### ISSUE THREAD

**Issue by christianlupus** [dmfs/opentasks#692]
When I reboot my phone (intentionally or by accident), the app starts after the restart all right but any reminder is gone. Also when I accidentally wiped the reminder to the right/left it is gone.
I think there are two things to be focused:
- [ ] Add option to the app to make the reminders more sticky. They should not be closable/wipable.
- [ ] The app should open any reminders that have started in the past (independent of the power state of the phone, the task has officially started) and are not yet marked as finished.
- [ ] optional: Add option to change behaviour if task notifications of past events should be evaluated at startup

**Comment by dmfs** [comment]
Version 1.1.18 now persists the notifications which are currently shown and restores them after an app update (except for the one to 1.1.18) or a reboot.
As said before, you can pin a task by opening it and tapping on the pin button. Once the plug-in concept is done you'll also be able to add a "Pin" option to the notification itself.
I'll close this one for now because the remaining issues will be handled in separate issues.
If you want the task above to be considered, please create a separate issue for each of them. Smaller issues have a higher chance of being considered for a release, hence it's a good idea to break them down to the smallest possible scope.
